src/components/Capture/capture.js

```diff
@@ -193,25 +194,46 @@ class Capture extends Component {
     }
   }
 
-  onApiError = (error) => {
-    this.setState({error, uploadInProgress: false})
+  onfidoErrorFieldMap = ([key, val]) => {
```

Comment:
not completely related to the scope of this, but since I had to hook to the triggering of every error (in order to track them), I took the opportunity to refactor it a bit.
The reason why I moved it here is because the error codes before where being mapped 50% in capture and 50% in the onfido api module. However, all of these final error codes are actually UI error codes, so they shouldn't be in a module which is not a UI module. This is why I moved it into capture.